src/features/locators/LocatorsPage.tsx

```diff
@@ -179,10 +185,31 @@ export const LocatorsPage = () => {
               ) : showSpinner ? (
                 <LocatorTreeSpinner />
               ) : (
-                <EmptyListInfo>
-                  Select the elements you need on the web page coverage and add them to the PO via a double-click or
-                  through the context menu
-                </EmptyListInfo>
+                <>
+                  <Modal
+                    title={EmptyListModal.Title}
+                    open={isEmptyListModalOpen}
+                    onCancel={() => setIsEmptyListModalOpen(false)}
+                    onOk={pageBack}
+                    okText={EmptyListModal.OkButtonTitle}
+                  >
+                    {EmptyListModal.Contents}
+                  </Modal>
+                  <EmptyListInfo>
+                    You can either create a{" "}
+                    <Button type="link" onClick={() => setIsEditModalOpen(true)}>
+                      Custom locator
+                    </Button>{" "}
+                    or refer to our{" "}
+                    <OnboardingPopup>
+                      <Button type="link" size="small">
+                        Onboarding tutorial
+                      </Button>
+                      .
+                    </OnboardingPopup>{" "}
+                    to understand the JDN features
+                  </EmptyListInfo>
```

Comment:
since `<EmptyListInfo>` became more complex, I'd suggest to create a new component for it and not to overload `LocatorsPage` with this jsx